core/src/main/java/org/infinispan/stats/impl/ClusterCacheStatsImpl.java

```diff
@@ -257,6 +270,29 @@ public long getMisses() {
       return getStatAsLong(MISSES);
    }
 
+   @ManagedAttribute(description = "Cluster wide approximate number of entries in the cache, including passivated entries and ignoring copies",
+         displayName = "Cluster wide approximate number of cache entries"
+   )
+   public long getApproximateEntries() {
+      return getStatAsLong(APPROXIMATE_ENTRIES);
+   }
+
+   @Override
+   @ManagedAttribute(description = "Cluster wide approximate number of entries in memory",
+         displayName = "Cluster wide approximate number of cache entries in memory"
+   )
+   public long getApproximateEntriesInMemory() {
+      return getStatAsLong(APPROXIMATE_ENTRIES_IN_MEMORY);
+   }
+
+   @Override
+   @ManagedAttribute(description = "Cluster approximate average number of unique entries",
+         displayName = "Cluster wide approximate number of cache entries (ignoring duplicate copies)"
+   )
+   public long getApproximateEntriesUnique() {
+      return getStatAsLong(APPROXIMATE_ENTRIES_UNIQUE);
+   }
+
    @ManagedAttribute(description = "Cluster wide total number of entries currently in the cache, including passivated entries",
          displayName = "Cluster wide total number of current cache entries"
```

Comment:
```suggestion
         displayName = "Cluster-wide total number of current cache entries"
```

Reply:
I can see in the following lines there are several more attributes. I can't seem to add suggestions there but I think it would be good to update them for consistency. In the display name we should go with "cluster-wide" with a hyphen for the compound adjective. In the description, use "across the cluster" at the end so the description uses different wording to the display name.